src/main/kotlin/kotlinx/kover/KoverPlugin.kt

```diff
@@ -28,10 +27,6 @@ class KoverPlugin : Plugin<Project> {
     private val defaultJacocoVersion = "0.8.7"
 
     override fun apply(target: Project) {
-        target.repositories.maven {
```

Comment:
Can't we add Maven Central here?

For the people who use company-wide mirrors, we can provide an option to opt-out, but it will work like a charm for everyone else. WDYT?

Reply:
I'm not sure if implicitly adding a repository is a good idea. Users are not notified about this in any way, and thus some may have a hidden problem with dependencies.

Of course, it is convenient for most users not to do additional actions, but at the same time, the vast majority of users have already added mavenCentral to the list of repositories. For example, in build.gradle.ts files should use the `maven Central' repository, because the `kotlin-scripting-compiler-embeddable` artifact is loaded from there.

Reply:
Thanks for the rationale! Let's use your approach then